1.10/s2i/bin/assemble

```diff
@@ -23,11 +23,11 @@ if [ -d ./nginx-default-cfg ]; then
   chmod -Rf g+rw ${NGINX_DEFAULT_CONF_PATH}
 fi
 
-if [ -d ./nginx-start-hook ]; then
+if [ -d ./nginx-start ]; then
   echo "---> Copying nginx start-hook scripts..."
```

Comment:
I'm thinking about `start-hook scrips` if it's misleading or not... Thought?

Reply:
I agree that it would be maybe a bit more precise, but what we really want to achieve here is to have the same names used in more images. And the `-hook` suffix is not used in other images, that's why it is prioritised. In the end, what is more important is a documentation of how the concrete hook is described.